### DISCUSSION TITLE
香香的邦布

### DISCUSSION THREAD

**Discussion by dick648** [share121/inter-knot#2215] [其他]
**市面上有给邦布添加特殊功能的吗，问问大家伙。**
![preview](https://github.com/user-attachments/assets/0dd52801-d39e-4614-b562-dc3f0d4050c2)

**Comment by xiguadabolio** [comment]
我感觉你要加的功能不正经

**Comment by dick648** [comment]
嘻嘻嘻<img src="https://bbs-static.miyoushe.com/static/2024/03/21/dab9bdd5012200a2902c8ae19dc1c8c9_4053703670259707891.png" alt="丽都漫步-可琳" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/03/21/dab9bdd5012200a2902c8ae19dc1c8c9_4053703670259707891.png" alt="丽都漫步-可琳" width="75" ><imgxixi src="https://bbs-static.miyoushe.com/static/2024/03/21/dab9bdd5012200a2902c8ae19dc1c8c9_4053703670259707891.png" alt="丽都漫步-可琳" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/03/21/dab9bdd5012200a2902c8ae19dc1c8c9_4053703670259707891.png" alt="丽都漫步-可琳" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/03/21/dab9bdd5012200a2902c8ae19dc1c8c9_4053703670259707891.png" alt="丽都漫步-可琳" width="75" >

**Comment by Yizakl** [comment]
可以给你安装自爆功能（）

**Comment by dean00100** [comment]
6分街有个改装店，店主是个机械臂的大叔，除了修理车辆也会接改装邦布的活，去他那里可能有戏，上次找他给邦布装的武器老带劲了！
<img src="https://bbs-static.miyoushe.com/static/2024/03/21/3d1687e289addc7611eba2c8585f4a9f_5782304423493346054.png" alt="丽都漫步-安比" width="75" >

**Comment by Phaethon-0019** [comment]
六分街有个叫RANDOM PLAY的录像店，旁边有个改装店，上次我托他改下邦布，老带劲了；
或者你去白祇重工，找他们的技术专家叫格莉丝，如果门口不放人，就跟他们说店长介绍来找格莉丝的，就是她可能会对你的邦布发癫，放心好了那是正常现象
